app/api/api.js

```diff
@@ -1,9 +1,11 @@
 /* eslint-disable global-require */
 import activitylogMiddleware from './activitylog/activitylogMiddleware';
+import headersMiddleware from './auth/headersMiddleware';
```

Comment:
@grafitto change the name to be more descriptive, headers is too generic, this middleware is here specifically to prevent CSRF attack

Reply:
How about `CSRFMiddleware`. I am not sure whether that is good 😂

Reply:
yes, definitely better